package.json

```diff
@@ -4,7 +4,7 @@
     "repository": "https://github.com/sulu/sulu.git",
     "scripts": {
         "preinstall": "node preinstall.js",
-        "lint:js": "eslint .",
+        "lint:js": "eslint . --cache",
```

Comment:
🎉 🎉 🎉 🎉 🎉 🎉 🎉

Reply:
I hope that does not affect the CI server... Otherwise I would be against it :see_no_evil: But I think we don't cache this file between runs there, so it should be fine.